src/index.ts

```diff
@@ -86,7 +87,6 @@ export class WorkOSSSOStrategy extends Strategy {
         return this.fail(err.text);
       }
 
-      this.error(err.text);
```

Comment:
This was getting auto-removed by vscode because the line is unreachable because of the `if(true)` above it.